### ISSUE TITLE
Markdown links don't work

### ISSUE THREAD

**Issue by PatrickAlphaC** [Cyfrin/aderyn#25]
This one works:

```
- [NC Issues](#nc-issues)
```

Because later in the markdown file, it's actually a markdown link:
```
# NC Issues
```

But this one doesn't work:
```
  - [NC-1: Functions not used internally could be marked external](#NC-1)
```

https://github.com/Cyfrin/aderyn/assets/54278053/ca52f0f1-09f0-4da9-b542-38f3e3465ce1

Because later it's not actually a markdown link:
```
<a name="NC-1"></a>
```

**Comment by aviggiano** [comment]
I'm also experiencing this problem.

See below the report produced by Aderyn. As a result, I can't see any of the issues flagged by the tool.

```markdown
# Aderyn Analysis Report

This report was generated by [Aderyn](https://github.com/Cyfrin/aderyn), a static analysis tool built by [Cyfrin](https://cyfrin.io), a blockchain security company. This report is not a substitute for manual audit or security review. It should not be relied upon for any purpose other than to assist in the identification of potential security vulnerabilities.
# Table of Contents

- [Summary](#summary)
  - [Files Summary](#files-summary)
  - [Files Details](#files-details)
  - [Issue Summary](#issue-summary)
- [Medium Issues](#medium-issues)
  - [M-1: Centralization Risk for trusted owners](#M-1)
  - [M-2: Using `ERC721::_mint()` can be dangerous](#M-2)
- [Low Issues](#low-issues)
  - [L-1: `abi.encodePacked()` should not be used with dynamic types when passing the result to a hash function such as `keccak256()`](#L-1)
  - [L-2: Unsafe ERC20 Operations should not be used](#L-2)
  - [L-3: Solidity pragma should be specific, not wide](#L-3)
- [NC Issues](#nc-issues)
  - [NC-1: Missing checks for `address(0)` when assigning values to address state variables](#NC-1)
  - [NC-2: Functions not used internally could be marked external](#NC-2)
  - [NC-3: Constants should be defined and used instead of literals](#NC-3)
  - [NC-4: Event is missing `indexed` fields](#NC-4)
  - [NC-5: The `nonReentrant` `modifier` should occur before all other modifiers](#NC-5)


# Summary

## Files Summary

| Key | Value |
| --- | --- |
| .sol Files | 111 |
| Total nSLOC | 8217 |


## Files Details

| Filepath | nSLOC |
| --- | --- |
| ... (repo is closed source) | ... (repo is closed source) |

```

**Comment by alexroan** [comment]
The fix will be released in v0.0.6. Thanks guys <3
